### ISSUE TITLE
Translation check and add (merge for the next minor release)

### ISSUE THREAD

**Issue by pengyin-shan** [portagenetwork/roadmap#137]
@robynnicholson : Please feel free to add the strings that are not translated in this issue. I'll check them and see if we need any code change to adopt the translation. If not, I'll just tag you in translation.io to upload translation then we are good to go. If so, I'll document it here and change the code. The code will be pushed to the next minor release together.

The following strings are confirmed to have a code-level change. If the item is checked, the changes are committed to the remote branch:
- [ ] "Click below to give data management staff at..."

Added by Omar in Slack:
- [ ] API Access
- [x] Allows the user to grant API access to organizations: translated
- [x] Grant API access: translated

**Comment by pengyin-shan** [comment]
@robynnicholson :

"Click below to give data management staff at.." and "API Access" is uploaded to dmp-assistant-test.

**Comment by robynnicholson** [comment]
Hi @pengyin-shan, thanks for opening this ticket. I've continued to track untranslated items in this Google doc: https://docs.google.com/document/d/1TLyFm_KA0QBBnLDcxcP21Q9Vg0JChlz-bvKQ0AU_JdA/edit?usp=sharing. 

I've updated the "Click below ..." and "API Access" translations in dmp-assistant-test. 

I'll also flag the following untranslated strings:
- From "Create a new Guidance Group" page of admin interface: "Name," "Published," and "Optional subset" (These strings do appear in translation.io and have translations added, however they still appear in English in UAT 3000, but ARE translated in prod)
![Screenshot 2022-02-15 161944](https://user-images.githubusercontent.com/46638411/154142753-5068e838-b172-4c99-a5cd-535633802cfb.png)
- From Usage Statistics page, both at top of page and "Apply your own filter" at the bottom: New Users, Total Users, New Plans, Total Plans - these strings are not available from translation.io, appear in English in BOTH UAT 3000 & prod, I have verified translations ready
![Screenshot 2022-02-15 164250](https://user-images.githubusercontent.com/46638411/154145853-120707bc-858c-4886-8bc3-fe8773c695e8.png)
